internal/cmd/test.go

```diff
@@ -179,7 +180,7 @@ func RunCommonCMDTests(test *testing.T, mainFunc func(), args []string, commonTe
 		return "", "", -1, false
 	}
 
-	if commonTestCase.ExpectedStdout != stdout {
+	if commonTestCase.ExpectedStdout != stdout && !commonTestCase.IgnoreStdout {
```

Comment:
Swap this ordering.
When you are short-circuiting and one check (e.g., an unknown-length string comparison) can be long, do short checks (e.g., boolean checks first).